test.js

```diff
@@ -11,4 +11,5 @@ test('main', t => {
 	t.is(humanizeString('-UnicornsAndRainbows'), 'Unicorns and rainbows');
 	t.is(humanizeString('  unicorns  and  rainbows  '), 'Unicorns and rainbows');
 	t.is(humanizeString('unicorns_and-Rainbows_andPonies  '), 'Unicorns and rainbows and ponies');
+	t.is(humanizeString('unicorns_and-Rainbows_andPonies', {originalCase: true}), 'Unicorns and Rainbows andPonies');
```

Comment:
```suggestion
	t.is(humanizeString('unicorns_and-Rainbows_andPonies', {originalCase: true}), 'Unicorns and Rainbows and Ponies');
```

This is what I expected.